modules/social_features/social_group/modules/social_group_request/social_group_request.module

```diff
@@ -299,29 +300,31 @@ function social_group_request_preprocess_activity(array &$variables) {
   $activity = $variables['elements']['#activity'];
 
   if (!$activity->get('field_activity_message')->isEmpty()) {
+    $message_storage = \Drupal::entityTypeManager()->getStorage('message');
     /** @var Drupal\message\Entity\Message $message */
```

Comment:
It is not necessary to have this annotation since bellow you have check on instance of 
`if ($message instanceof Message) {`